packages/devtools_app/lib/src/analytics/_analytics_stub.dart

```diff
@@ -42,11 +41,7 @@ void timeSync(
 }) {
   // Execute the operation here so that the desktop app still functions without
   // the real analytics call.
-  try {
-    syncOperation();
-  } on ProcessCancelledException catch (_) {
-    // Do nothing for instances of [ProcessCancelledException].
-  }
+  syncOperation();
```

Comment:
we need to catch and rethrow here and below if we want the consumer of timeSync and timeAsync to be able to handle the exceptions

Reply:
discussed offline and agreed it is ok to do nothing